plugins/drivers/driver.go

```diff
@@ -314,6 +314,12 @@ type MountConfig struct {
 	Readonly bool
 }
 
+func (m *MountConfig) IsEqual(o *MountConfig) bool {
+	return m.TaskPath == o.TaskPath &&
+		m.HostPath == o.HostPath &&
+		m.Readonly == o.Readonly
```

Comment:
Hmm - good spot. It's actually going to be a bit tricky to validate this, bc of intersecting paths. I think non-duplicate declarations will be a problem for future us though - because they'll needed to be handled at a different level of abstraction (namely bc when we add CSI volume mounting, there will probably be a different hook managing them)